vendor-extra/JatsContentBundle/src/Resources/config/services.xml

```diff
@@ -14,12 +14,36 @@
             <tag name="libero.content_page.handler" alias="jats"/>
         </service>
 
-        <service id="Libero\JatsContentBundle\ViewConverter\ArticleTitleHeadingVisitor"
-            class="Libero\JatsContentBundle\ViewConverter\ArticleTitleHeadingVisitor">
+        <service id="Libero\JatsContentBundle\ViewConverter\HeadingVisitor"
+            class="Libero\JatsContentBundle\ViewConverter\HeadingVisitor">
             <argument type="service" id="Libero\ViewsBundle\Views\ViewConverter"/>
             <tag name="libero.view_converter"/>
         </service>
 
+        <service id="Libero\JatsContentBundle\ViewConverter\LinkVisitor"
+            class="Libero\JatsContentBundle\ViewConverter\LinkVisitor">
+            <argument type="service" id="Libero\ViewsBundle\Views\ViewConverter"/>
+            <tag name="libero.view_converter" priority="-10"/>
+        </service>
+
+        <service id="Libero\JatsContentBundle\ViewConverter\FrontItemTagsVisitor"
+            class="Libero\JatsContentBundle\ViewConverter\FrontItemTagsVisitor">
+            <argument type="service" id="Libero\ViewsBundle\Views\ViewConverter"/>
+            <tag name="libero.view_converter"/>
+        </service>
+
+        <service id="Libero\JatsContentBundle\ViewConverter\KeywordGroupTagListVisitor"
+            class="Libero\JatsContentBundle\ViewConverter\KeywordGroupTagListVisitor">
+            <argument type="service" id="Libero\ViewsBundle\Views\ViewConverter"/>
+            <argument type="service" id="translator"/>
+            <argument type="collection">
+                <!-- Map of @kwd-group-type values to translation keys -->
+                <argument key="author-keywords">libero.jats.keyword_group.title.author_keywords</argument>
```

Comment:
are these names (`author-keywords`, `research-organism`) generic or are they supposed to be overridden via configuration?

Reply:
Default ones that Libero understands (part of the higher-level question, _what_ JATS are we supporting directly); can provide configuration to add in custom ones (doable as it stands, but requires code).